pagerduty_handler.json

```diff
@@ -0,0 +1,8 @@
+{
+  "handlers": {
+    "pagerduty": {
+      "type": "pipe",
+      "command": "/usr/share/pdagent-integrations/bin/pd-sensu -k YOUR-INTEGRATION-KEY-HERE"
+    }
+  }
+}
```

Comment:
could also be `sensu-extra` or something like that.

i would normally ignore until we have multiple files, but the url for this file is going to be used in the integration guide so i'd prefer to not have to change it later